### ISSUE TITLE
Change "Record audit" in English

### ISSUE THREAD

**Issue by jenniferward** [rism-ch/muscat#569]
Please change the terminology in the section "Record audit"

Record audit --> please change to: Record origin
Full level encoding with examination --> Full record, material examined
Abbreviated level encoding with examination --> Brief record, material examined
Retro-cataloguing --> Retrocataloging
Imported encoding --> Imported record

**Comment by lpugin** [comment]
I am not really convinced that "Record source" is the best choice. The "source" term can be misleading. I would suggest "Record level", or "Record provenance", or "Record origin" but I would avoid "source".

**Comment by jenniferward** [comment]
OK. "Record origin" is better. 

I will be asking Barbara Wiermann what her intentions are here, because she is combining "provenance" and "level" here and I'm not convinced that's a good idea in every case.

**Comment by lpugin** [comment]
> I will be asking Barbara Wiermann what her intentions are here, because she is combining "provenance" and "level" here and I'm not convinced that's a good idea in every case.

I fully agree with you and I already emitted the same reserve. I would recommend to focus on the record level here. The provenance should be recorded separately.

**Comment by xhero** [comment]
What does "Retrocataloging" mean? Should I just leave it in Italian and French?

**Comment by jenniferward** [comment]
Retrocataloging is just cataloging old materials that are in old (card) catalogs but not in OPACs. Now I wonder if by "retrocataloging" she doesn't actually mean "retrospective conversion" -- when you take analog records like library catalog cards, scan them, and import them into a catalog. It's also what we're doing with the B/I and B/II data and it would be a useful distinction to make.

Let's wait on this until I hear back from BW.

**Comment by xhero** [comment]
It is “Retroeinspielung” in German

**Comment by xhero** [comment]
How should we move on with this? I still have "Retrocataloging" in French and Italian...

**Comment by jenniferward** [comment]
We asked Barbara yesterday and I don't know when I will hear back. When is the last day to change this before the release?

**Comment by HirschSt** [comment]
3.7 will be closed today, but we can adjust terminology and labels with an early 3.7.1

**Comment by jenniferward** [comment]
Ok. I see it's changed on beta, thanks! I have no guidelines for this field, though.

**Comment by HirschSt** [comment]
Probably the field would be set to readonly with initial value "full record", so we can postpone the discussion about definition, concepts, terminology etc. leading to "simple" guidelines

**Comment by HirschSt** [comment]
Done with b7c9a73d95d6cddb30e952367eaf8d3d24a726e4
